frontend/client/styles/components/footer.scss

```diff
@@ -2,34 +2,61 @@
 
 #footer {
   display: flex;
+  // `margin-top: auto;` pins the footer to the bottom of the app, see https://stackoverflow.com/a/33924704
+  margin-top: auto;
 }
 
 .footer-not-login {
-  padding-right: 101px;
+  @include media('>phone') {
+    padding-right: 101px;
```

Comment:
Same thing as the navbar being pushed over.

Reply:
See https://github.com/covidwatchorg/portal/pull/454#discussion_r467124279